### DISCUSSION TITLE
Is there a reason why 1.25 speed is not available on Linux?

### DISCUSSION THREAD

**Discussion by Salvaju29ro** [smplayer-dev/smplayer#166] [Q&A]
Hi, I noticed that the shortcut to increase the speed to 1.25 is not available on Linux, is there any particular reason?

I use the Arch Linux package, I don't know if anything changes

**Linux**:
![Linux](https://user-images.githubusercontent.com/10899871/127622098-6068bf17-2e12-4b81-9d4e-e8d15391d9fb.png)

**Windows**:
![Windows](https://user-images.githubusercontent.com/10899871/127622112-d958519b-adda-4976-a93c-b63116cb37d3.png)

**Comment by smplayer-dev** [comment]
You're probably using an older version on Linux. That option was added after the 21.1 release, so you need a development version. I guess you can try the appimage or flatpak version.

https://github.com/smplayer-dev/smplayer/releases/tag/v21.1.0.9823

**Reply by Salvaju29ro** [reply]
> You're probably using an older version on Linux. That option was added after the 21.1 release, so you need a development version. I guess you can try the appimage or flatpak version.

Thanks for the reply
If it is added in a future stable release, I can also wait. For now I can use 10 + 10 + 5 to get to 1.25
